KeyboardConstraint/KeyboardAdjustConstraint.m

```diff
@@ -89,4 +83,36 @@ - (void)_keyboardDidChangeVisible:(NSNotification *)notification
     }
 }
 
+// MARK - Private helpers
+
+- (CGFloat)layoutGuideHeight {
+    id layoutGuide = [self layoutGuideItem];
+    
+    if ([layoutGuide conformsToProtocol:@protocol(UILayoutSupport)]) {
+        // The layout guide is a regular bottom layout guide
+        return ((id<UILayoutSupport>) layoutGuide).length;
+    }
+    
+    if ([layoutGuide isKindOfClass:[UILayoutGuide class]] && @available(iOS 11, *)) {
```

Comment:
Not massively big deal. But compiler complains about safe guarding this. 
Might have to break info double if statements to shut it up:
```objc
if (@available(iOS 11.0, *))  {
    if ([layoutGuide isKindOfClass:[UILayoutGuide class]]) {
        return ((UILayoutGuide *) layoutGuide).owningView.safeAreaInsets.bottom;
    }
}
```

Reply:
Good call, I had disabled warnings on pods so I didn't notice this. I'll fix that 👍